pom.xml

```diff
@@ -54,6 +54,12 @@
             <version>1.7.30</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-simple</artifactId>
+            <version>1.7.30</version>
+        </dependency>
+
```

Comment:
Should not include an implementation of slf4j, this will cause the user of this library to lose control of the log library they want( e.g. log4j, logback)